### ISSUE TITLE
Add react-contextual example

### ISSUE THREAD

**Issue by juandc** [GantMan/ReactStateMuseum#56]
[React Contextual](https://github.com/drcmda/react-contextual) works with the react context API. Here a detailed explanation: https://github.com/drcmda/react-contextual/blob/master/PITFALLS.md

**Comment by erdahuja** [comment]
Hi, I want to contribute. New to open source, Pretty good in js, How can I begin? @GantMan

**Comment by erdahuja** [comment]
Yes git pro, react really good, been a js developer for 2 years now.
Used the listed ones few. will see if I come across an.
No don't know react > react native.

**Comment by GantMan** [comment]
Basically, you run `react-native init ProjectNameHere` and then when it's done you `cd` into the folder and run `react-native run-ios` on a mac.

If you can get that basic app up, you can figure everything else out.
